releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,6 +4,9 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v8.0.0 - July, 3, 2024
+- Updated the task to support Node.js 20. It now requires either Node.js 16 or Node.js 20, whereas previously it required Node.js 16.
```

Comment:
Does this sound more positive?

Added support for Node.js 20. This pipeline task now supports both Node.js 16 and Node.js 20.

Reply:
> Added support for Node.js 20. This pipeline task now supports both Node.js 16 and Node.js 20.

Yes this does sound more positive

Reply:
This has been updated, thanks